spec/lib/expansion_rules_spec.rb

```diff
@@ -47,6 +47,7 @@
     let(:mainstream_browser_page_fields) { default_fields + %i[description] }
     let(:need_fields) { default_fields + [%i[details role], %i[details goal], %i[details benefit], %i[details met_when], %i[details justifications]] }
     let(:finder_fields) { default_fields + [%i[details facets]] }
+    let(:ministerial_role_fields) { default_fields + [%i[details body], %i[details seniority], %i[details role_payment_type]] }
```

Comment:
Or an alternative to my comment above, we could this define this as:

```suggestion
    let(:ministerial_role_fields) { role_fields + [%i[details seniority]] }
```

I don't mind which one, it does mean the variables won't be in alphabetical order, but I think that's okay.